doc/intro.md

```diff
@@ -378,12 +376,16 @@ A block is simply built from:
 * A block header.
 * The list of inputs remaining after cut-through.
 * The list of outputs remaining after cut-through.
-* A single kernel offset to cover the full block.
+* A single kernel offset (sum of all kernel offsets) to cover the full block.
 * The transaction kernels containing, for each transaction:
-  * The public key `(r-a)*G`.
-  * The signatures generated using the excess value.
+  * The public key `(r-a)*G`, which is the (modified) kernel excess.
+  * The signatures generated using the (modified) excess value `(r-a)` as the private (signing) key.
   * The mining fee.
 
+The block contents satisfy: 
+
+sum(outputs) - sum(inputs) = sum(kernel_excess) + kernel_offset*G
```

Comment:
Each block also increases the total supply by 60 grin from the coinbase reward.

Reply:
It was mentioned in the previous comment (https://github.com/mimblewimble/grin/pull/3281#issuecomment-605707368) as well, but the intro doesn't discuss this at all. I find it better not get into this at this introductory level.

Reply:
It was mentioned in the previous comment (https://github.com/mimblewimble/grin/pull/3281#issuecomment-605707368) as well, but the intro doesn't discuss this at all. I find it better not get into this at this introductory level.